packages/client/wallets/aa/src/utils/signer.ts

```diff
@@ -10,8 +10,9 @@ import {
     getUrlProviderByBlockchain,
     getWeb3AuthBlockchain,
 } from "@/blockchain";
-import { FireblocksNCWSigner, UserIdentifier, WalletConfig, Web3AuthSigner } from "@/types";
+import { FireblocksNCWSigner, UserIdentifier, UserIdentifierParams, WalletConfig, Web3AuthSigner } from "@/types";
```

Comment:
I don't see `UserIdentifierParams` being used anywhere, can we remove?
```suggestion
import { FireblocksNCWSigner, UserIdentifier, WalletConfig, Web3AuthSigner } from "@/types";
```